components/forms/CreateTxForm/MsgForm/MsgClaimRewardsForm.tsx

```diff
@@ -40,12 +41,14 @@ const MsgClaimRewardsForm = ({
           return false;
         }
 
-        return isTxMsgClaimRewards(msg);
+        return true;
       };
 
-      const msg: TxMsgClaimRewards = {
-        typeUrl: "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward",
-        value: { delegatorAddress, validatorAddress },
+      const msgValue: MsgWithdrawDelegatorReward = { delegatorAddress, validatorAddress };
```

Comment:
Here it does not make much difference. But in general it is better to construct those messages from fromPartial since it allows you to skip fields you don't need:

```suggestion
      const msgValue = MsgWithdrawDelegatorReward.fromPartial({ delegatorAddress, validatorAddress });
```